draft-ietf-mmusic-mux-exclusive.xml

```diff
@@ -201,31 +205,30 @@
+                RTP/RTCP multiplexing, the answerer MUST associate an SDP 'rtcp-mux' attribute with
+                the corresponding SDP media description ("m=") in the associated answer, following the procedures in
+                <xref target="RFC5761" pageno="false" format="default"/>. If the answerer does not
                 accept the usage of RTP/RTCP multiplexing, the answerer MUST either reject the SDP media description ("m=")
                 by setting the port value to zero in the associated answer, or reject the whole offer,
                 following the procedures in <xref target="RFC3264" pageno="false" format="default"/>.
             </t>
             <t>
-                In addition, if the answerer associates an SDP 'rtcp-mux-only' attribute with
-                an SDP media description ("m=" line) in the answer, and if the corresponding "m=" line
-                in the associated offer contained an SDP 'rtcp-mux' attribute, the answerer MUST in addition associate
-                an SDP 'rtcp-mux' attribute with the same "m=" line, following the procedures in
-                <xref target="RFC5761" pageno="false" format="default"/>.
+                The answerer MUST NOT associate an SDP 'rtcp-mux-only' attribute with an
+                SDP media description ("m=" line) in the answer.
             </t>
         </section>
         <section title="Offerer Processing of the SDP Answer" anchor="sec-of-off-ans">
             <t>
-                If an offerer associated an SDP 'rtcp-mux-only' attribute with an RTP-based SDP media description ("m=" line)
-                in an offer, and if the corresponding SDP media description ("m=" line) in the associated answer contains
-                an SDP 'rtcp-mux-only' attribute, and/or an SDP 'rtcp-mux' attribute, the offerer MUST apply the
-                RTP/RTCP multiplexing procedures <xref target="RFC5761" pageno="false" format="default"/>
-                to the associated RTP-based media. If the corresponding SDP media description ("m=" line) in the associated
-                answer does not contain an SDP 'rtcp-mux-only' attribute, nor an SDP 'rtcp-mux' attribute,
-                the offerer MUST either take appropriate actions in order to disable the associated RTP-based media,
-                or send a new offer without associating an SDP 'rtcp-mux-only' attribute with the
-                SDP media description ("m=" line).
+                If an offerer associated an SDP 'rtcp-mux-only' attribute with an RTP-based
+                SDP media description ("m=" line) in an offer, and if the corresponding
+                SDP media description ("m=" line) in the associated answer contains
+                an SDP 'rtcp-mux' attribute, the offerer MUST apply the RTP/RTCP multiplexing
+                procedures <xref target="RFC5761" pageno="false" format="default"/>
+                to the associated RTP-based media. If the corresponding SDP media description
```

Comment:
Sure, but since the offer contains both a=rtcp-mux-only and a=rtcp-mux, I think it's good to keep the text. The text does refer RFC 5761, so...